apps/src/templates/studioHomepages/CourseBlocksGradeBands.story.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import CourseBlocksGradeBands from './CourseBlocksGradeBands';
+
+export default storybook => {
+  return storybook
```

Comment:
Nit: You can remove a layer of indentation from this file if you use an arrow function expression:

```jsx
export default storybook => storybook
  .storiesOf('CourseBlocksGradeBands', module)
  .addStoryTable([
    //...
  ]);
```